scss/common.scss

```diff
@@ -84,30 +83,8 @@ body {
     top: 50%;
     left: 50%;
   }
-  // .bx--select--inline{
-  //   svg.bx--select__arrow{
-  //     position: absolute
-  //   }
-  // }
-}
-
-#toggle-perspective {
```

Comment:
does removing this not mess up the style for the perspective switcher?

Reply:
will remove in future clean up pr

Reply:
I didn't know this is for toggle-perspective switcher. Shouldn't it be in header instead?

Reply:
this is in header, it's just that something that I couldn't find in the grc style was overriding it so I added that to fix it. If the perspective switcher looks ok after removing it maybe it's not necessary though

Reply:
I updated the patternfly css to latest so in theory we should no longer need to override css. Can you test it again after the pr is merged? It it doesn’t work, we can add css back

Reply:
It could be also carbon components overriding the css. How can I test it?